hypothesis-python/src/hypothesis/core.py

```diff
@@ -387,6 +387,24 @@ def skip_exceptions_to_reraise():
 exceptions_to_reraise = skip_exceptions_to_reraise()
 
 
+def failure_exceptions_to_catch():
+    """Return a tuple of exceptions meaning 'this test has failed', to catch.
+
+    This is intended to cover most common test runners; if you would
+    like another to be added please open an issue or pull request.
+    """
+    exceptions = [Exception]
+    try:  # pragma: no cover
+        from _pytest.outcomes import OutcomeException
+        exceptions.append(OutcomeException)
+    except ImportError:
+        pass
+    return tuple(exceptions)
+
+
+exceptions_meaning_failure = failure_exceptions_to_catch()
```

Comment:
Top level constants should be upper case (I wonder if we can add a linter rule for that?)

Reply:
Matching the block above again, but for this I'll fix both.